client/my-sites/subscribers/components/subscriber-totals/index.tsx

```diff
@@ -75,28 +94,28 @@ const SubscriberTotals: React.FC< SubscriberTotalsProps > = ( {
 				<>
 					<span className="subscriber-totals__filtered-count">
 						{ searchTerm
-							? translate( '%(count)d matching result', '%(count)d matching results', {
-									count: filteredCount,
-									args: { count: filteredCount },
+							? translate( '%(count)s matching result', '%(count)s matching results', {
```

Comment:
The updated text risks being interpreted as fuzzy, which will result in a string being injected for an integer. The way to get around this is to rename the variables to a much longer name to force a lower similarity of the two strings.

There is a utility to compare strings that you can use to confirm: https://mc.a8c.com/i18n/tools/gp-string-similarity.php?string_a=%25%28count%29d+matching+result&string_b=%25%28count%29s+matching+result 

With the above change, the similarity is 0.964, which will flag the translation as fuzzy and may lead to issues. The updated string should be below 0.8 to avoid.

Reply:
Thanks! I renamed the variables and checked that each similarity score was under 0.8.